readthedocs/projects/tasks.py

```diff
@@ -489,7 +489,7 @@ def run_setup(self, record=True):
             # triggered before the previous one has finished (e.g. two webhooks,
             # one after the other)
             if not isinstance(self.setup_env.failure, VersionLockedError):
-                self.send_notifications(self.version.pk, self.build['id'])
+                self.send_notifications(self.version.pk, self.build['id'], email=True)
```

Comment:
I think we don't want to send an email when the version is locked. It will be automatically re-triggered in five minutes.

Reply:
I believe we were sending email for this before thats why I added it as `email=True`.
Because when ever `self.send_notifications()` was called an email would be sent.

Reply:
OK. Let's do not change the behavior in this PR. Although, I think it's not correct to send an email communicating the version is locked. I think it's useless.